src/Doctrine/Odm/Filter/BooleanFilter.php

```diff
@@ -139,4 +144,27 @@ protected function filterProperty(string $property, $value, Builder $aggregation
 
         $aggregationBuilder->match()->field($matchField)->equals($value);
     }
+
+    /**
+     * @return array<string, string>
+     */
+    public function getSchema(Parameter $parameter): array
+    {
+        return $parameter->getSchema() ?? ['type' => 'boolean'];
+    }
+
+    public function getOpenApiParameters(Parameter $parameter): OpenApiParameter|array|null
+    {
+        $in = $parameter instanceof QueryParameter ? 'query' : 'header';
+        $key = $parameter->getKey();
+
+        return [
+            new OpenApiParameter(
+                name: $key,
+                in: $in,
+                required: false,
+                schema: ['type' => 'boolean'],
+            ),
+        ];
```

Comment:
in fact let's not implement this here. We need this only for filters like the DateFilter that adds keys like `date[before]=`

Reply:
What does that mean? In what way should I remove it? And just for Odm or also for Orm?